docs/storybook/src/widget/WidgetContentLayout/WidgetContentLayout.Header.stories.tsx

```diff
@@ -0,0 +1,242 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import type { Meta, StoryObj } from "@storybook/react-vite";
+import {
+  AppUiDecorator,
+  ConsoleToActionsDecorator,
+  InitializerDecorator,
+} from "../../Decorators";
+import { Page } from "../../AppUiStory";
+import { WidgetContentLayoutHeader } from "./WidgetContentLayout.Header";
+import { SvgPlaceholder } from "@itwin/itwinui-icons-react";
+import { Button } from "@itwin/itwinui-react";
+
+const meta = {
+  title: "Widget/Widget Content Layout/WidgetContentLayout.Header",
+  component: WidgetContentLayoutHeader,
+  tags: ["autodocs"],
+  decorators: [InitializerDecorator, ConsoleToActionsDecorator, AppUiDecorator],
+  parameters: {
+    docs: {
+      page: () => <Page />,
+      source: {
+        code: `<WidgetContentLayout id="header-widget">
+    <WidgetContentLayout.Header
+      title="Title"
+      menu={{
+        title: "Dropdown Menu",
+        items: [
+          { label: "Item 1", onClick: () => console.log("Item 1 clicked") },
+          { label: "Item 2", onClick: () => console.log("Item 2 clicked") },
+        ],
+      }}
+      icons={[
+        { label: "Icon 1", icon: <SvgPlaceholder />, onClick: () => console.log("Icon 1 clicked") },
+        { label: "Icon 2", icon: <SvgPlaceholder />, onClick: () => console.log("Icon 2 clicked") },
+      ]}
+      buttons={[
+        <Button>Button 1</Button>,
+        <Button>Button 2</Button>
+      ]}
+    />
+    <WidgetContentLayout.Content>Content</WidgetContentLayout.Content>
+    <WidgetContentLayout.Footer>Footer</WidgetContentLayout.Footer>
+  </WidgetContentLayout>`,
+      },
+    },
+  },
+} as Meta<typeof WidgetContentLayoutHeader>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Full: Story = {
+  args: {
+    title: "Title",
+    menu: {
+      title: "Select what is displayed below",
+      items: [
+        { label: "Item 1", onClick: () => console.log("Item 1 clicked") },
```

Comment:
Why use `console`, when https://storybook.js.org/docs/essentials/actions are available and can be used directly